packaging/lwnode.spec

```
@@ -158,16 +166,18 @@ rm -rf %{buildroot}
 mkdir -p %{buildroot}%{_bindir}
 mkdir -p %{buildroot}%{_libdir}
 
+%define target_lib %{target_src}/lib
+
 rm -f %{target_lib}/*.tmp %{target_lib}/*.TOC
 
 %if 0%{?static_escargot} == 0
   cp %{target_src}/gen/escargot/libescargot.so %{buildroot}%{_libdir}
-  strip -v -g %{target_src}/gen/escargot/libescargot.so
 %endif
 %if "%{lib_type}" == "shared"
   cp %{target_lib}/liblwnode.so* %{buildroot}%{_libdir}
-  strip -v -g %{target_lib}/liblwnode.so*
 %endif
+find ./out/tizen/ -name "*.so*" -type f -exec strip -v -g {} \;
```

```suggestion
find out/tizen -name "*.so*" -type f -exec strip -v -g {} \;
```

or

`find out/tizen -name "*.so*" -type f -exec strip -v -g {} +`


`+` gathers as many filenames as possible and passes them as arguments to the strip command at once (for example, strip -v -g escargot.so lwnode.so...).

---

Done.